StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetDeferredValidator.swift

```diff
@@ -53,6 +54,8 @@ struct PaymentSheetDeferredValidator {
             return
         }
         guard intentPaymentMethod.stripeId == paymentMethod.stripeId else {
+            let errorAnalytic = ErrorAnalytic(event: .paymentSheetIntentPaymentMethodIdMismatch, error: PaymentSheetError.unknown(debugDescription: "Payment method ids don't match"))
```

Comment:
Let's use the same error message in `debugDescription` as we do in the message below in the error we throw.